src/Chapter4.hs

```diff
@@ -628,6 +672,9 @@ Can you implement a monad version of AND, polymorphic over any monad?
 
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
+
+-- Will skip the Challenge Tasks for now and come back to them later
```

Comment:
No worries! Let me know if you would like any guidance or hints on these two 👌🏼

Reply:
I will be solving these problems at my leisure soon and ask for your help if stuck ! Thank you!